src/io/gaslib.jl

```diff
@@ -348,7 +348,7 @@ function _get_junction_entry(junction)
     lon_sym = :geoWGS84Long in keys(junction) ? :geoWGS84Long : :y
     lon = parse(Float64, junction[lon_sym])
 
-    height = parse(Float64, junction["height"][:value])
+    elevation = parse(Float64, junction["height"][:value])
```

Comment:
should not `junction["height"][:value]` be `junction["elevation"][:value]` instead?

Reply:
Height is a gaslib property. So I converted to elevation only after reading it.